vm/ubpf_jit_x86_64.c

```diff
@@ -938,6 +943,24 @@ resolve_patchable_relatives(struct jit_state* state)
         memcpy(offset_ptr, &rel, sizeof(uint32_t));
     }
 
+    for (i = 0; i < state->num_local_calls; i++) {
+        struct patchable_relative local_call = state->local_calls[i];
+
+        int target_loc;
+        assert(local_call.target_offset == 0);
+        assert(local_call.target_pc != TARGET_PC_EXIT);
+        assert(local_call.target_pc != TARGET_PC_RETPOLINE);
+
+        target_loc = state->pc_locs[local_call.target_pc];
+
+        /* Assumes call offset is at end of instruction */
+        uint32_t rel = target_loc - (local_call.offset_loc + sizeof(uint32_t));
+        rel -= 7; // For the "sub rsp, 8" instruction that is inserted before the call to align the stack.
```

Comment:
I hope that this doesn't make you mad, but could we supplement your excellent comment here with a named constant for the 7? That way we could change it later if there is a situation where the code for the stack alignment gets more complicated for some reason?

Reply:
Thanks for catching this. I had meant to fix it, but forgot. It's now dynamically computed for both x64 and arm64.